components/sync15/src/request.rs

```diff
@@ -331,57 +338,61 @@ where
             serde_json::to_writer(&mut self.queued, &record).unwrap();
         }
 
+        // Add the record to the queue tracker
         self.post_limits.record_added(payload_length);
         self.batch_limits.record_added(payload_length);
 
         Ok(true)
     }
 
+
+    /// Returns batchid 
```

Comment:
Remove comment or make more precise (e.g., move the comment on line 352 up here).